store/src/java-test/com/zimbra/cs/imap/MemcachedImapCacheTest.java

```diff
@@ -54,7 +54,7 @@ public class MemcachedImapCacheTest {
      */
     @Before
     public void setUp() throws Exception {
-        MailboxTestUtil.initProvisioning("store/");
+        MailboxTestUtil.initProvisioning(MailboxTestUtil.getZimbraServerDir(null));
```

Comment:
What does it internally uses? `MailboxTestUtil.initProvisioning(MailboxTestUtil.getZimbraServerDir(null));`